onnxruntime/core/providers/cpu/nn/conv_impl.h

```diff
@@ -135,7 +135,7 @@ Status Conv<T>::Compute(OpKernelContext* context) const {
             col_buffer_data,
             &CPUMathUtil::Instance());
       } else {
-        math::Im2colNd<T, CPUMathUtil, StorageOrder::NCHW>(
+        math::Im2colNd<T, CPUMathUtil, StorageOrder::NCHW>()(
```

Comment:
If you wanted to remove the extra () you could change the operator() into the constructor, since it doesn't return a value it can work. Then it'll be an object that really looks just like a function.